### ISSUE TITLE
Error with Auto Login URL

### ISSUE THREAD

**Issue by ruNN0r** [linuxserver/Heimdall#948]
Hello, I use Heimdall with 3 accounts.
When I try to log in with the Auto Login URL, it works for the time being. After a while, however, the error below is displayed. I have tried the following:
- Auto Login URL set as start page 
- Used Auto Login and then the default URL.

As soon as I delete the cookies, it works again for the time being. 
It is also independent of whether the browser was closed. So the session ID should still be the same.

```
Error thrown with message "Call to a member function setCookie() on null"
Stacktrace:
#34 Error in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php:191
#33 Illuminate\Foundation\Http\Middleware\VerifyCsrfToken:addCookieToResponse in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php:80
#32 Illuminate\Foundation\Http\Middleware\VerifyCsrfToken:Illuminate\Foundation\Http\Middleware\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Support/helpers.php:433
#31 tap in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php:82
#30 Illuminate\Foundation\Http\Middleware\VerifyCsrfToken:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
#29 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php:49
#28 Illuminate\View\Middleware\ShareErrorsFromSession:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
#27 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php:116
#26 Illuminate\Session\Middleware\StartSession:handleStatefulRequest in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php:62
#25 Illuminate\Session\Middleware\StartSession:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
#24 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php:37
#23 Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
#22 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php:67
#21 Illuminate\Cookie\Middleware\EncryptCookies:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
#20 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:103
#19 Illuminate\Pipeline\Pipeline:then in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Router.php:687
#18 Illuminate\Routing\Router:runRouteWithinStack in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Router.php:662
#17 Illuminate\Routing\Router:runRoute in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Router.php:628
#16 Illuminate\Routing\Router:dispatchToRoute in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Routing/Router.php:617
#15 Illuminate\Routing\Router:dispatch in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:165
#14 Illuminate\Foundation\Http\Kernel:Illuminate\Foundation\Http\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:128
#13 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/fideloper/proxy/src/TrustProxies.php:57
#12 Fideloper\Proxy\TrustProxies:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
#11 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php:21
#10 Illuminate\Foundation\Http\Middleware\TransformsRequest:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
#9 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php:21
#8 Illuminate\Foundation\Http\Middleware\TransformsRequest:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
#7 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php:27
#6 Illuminate\Foundation\Http\Middleware\ValidatePostSize:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
#5 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php:63
#4 Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode:handle in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:167
#3 Illuminate\Pipeline\Pipeline:Illuminate\Pipeline\{closure} in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:103
#2 Illuminate\Pipeline\Pipeline:then in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:140
#1 Illuminate\Foundation\Http\Kernel:sendRequestThroughRouter in /var/www/localhost/heimdall/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:109
#0 Illuminate\Foundation\Http\Kernel:handle in /var/www/localhost/heimdall/public/index.php:55

```

**Comment by keriati** [comment]
Could be something with **app/Http/Middleware/VerifyCsrfToken.php**

**Comment by KodeStar** [comment]
Duplicate of #379

**Comment by KodeStar** [comment]
This has been fixed in 2.5.0, see #379 for more details
